data/styles/clear/include/Basemap.mapcss

```diff
@@ -577,10 +578,16 @@ area|z15-[landuse=churchyard]
 area|z10-[landuse=farmland],
 {fill-position: background; fill-color: @farmland; fill-opacity: 1;}
 
+
 /* Parking */
 
 area|z15-[amenity=parking]
 {fill-color: @parking;}
+line|z18-[amenity=parking_space]
+{fill-position: background;width: 2;color: @white;opacity: 0.3;}
```

Comment:
fill-position: background; is useless here, as parking_space become linear.

Reply:
fixed